postman/Get Broadcast Details.postman_collection.json

```diff
@@ -0,0 +1,525 @@
+						"id": "028258ae-2551-44e3-a6c3-40f3f93d68e6",
+						"exec": [
+							"pm.test(\"Successfully retrieved the number of draft broadcasts on list\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.a('number')",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=draft",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts",
+						"total"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "draft"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Broadcast Totals  - Scheduled",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "aebe59b2-ac5a-41af-9f56-21bd56794e16",
+						"exec": [
+							"pm.test(\"Successfully retrieved the number of scheduled broadcasts on list\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.a('number')",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=scheduled",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"broadcasts",
+						"total"
+					],
+					"query": [
+						{
+							"key": "status",
+							"value": "scheduled"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Broadcast Totals  - Sent",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "96754c67-4f3b-4523-9f57-27cf07835971",
+						"exec": [
+							"pm.test(\"Successfully retrieved the number of sent broadcasts on list\", function(){",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.total_size).to.be.a('number')",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://api.aweber.com/1.0/accounts/{{accountId}}/lists/{{listId}}/broadcasts/total?status=sent",
+					"protocol": "https",
+					"host": [
+						"api",
+						"aweber",
+						"com"
+					],
+					"path": [
+						"1.0",
+						"accounts",
+						"{{accountId}}",
```

Comment:
variables should follow a single language paradigm either python `account_id` or php `accountId` that apply to all variables for consistency.